store/rootmulti/merkle_map.go

```diff
@@ -30,13 +29,9 @@ func newMerkleMap() *merkleMap {
 func (sm *merkleMap) set(key string, value []byte) {
 	sm.sorted = false
 
-	// The value is hashed, so you can check for equality with a cached value (say)
```

Comment:
Okay, this is the same change as in tendermint. Again, why?

(Also this breaks ics23-tendermint, which needs a PR to update the format and regenerate the proofs)

Reply:
https://github.com/confio/ics23-tendermint/pulls if you really want to make this.

But there should be no real reason to mix it in this PR.

Reply:
Why do we have this code in two places in the first place?